docs/quickstarts/packet.md

```diff
@@ -150,90 +150,89 @@ lokomotive-demo-pool-1-worker-1    True     KubeletReady    kubelet is posting r
-to learn about its usage.
+At this point you should have access to a Lokomotive cluster and can use it to deploy applications.
+
+If you don't have any Kubernetes experience, you can check out the [Kubernetes
+Basics](https://kubernetes.io/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/) tutorial.
 
-**Note**: Lokomotive sets up a pretty restrictive Pod Security Policy that
-disallows running containers as root by default, check the [Pod Security Policy
-documentation](../concepts/securing-lokomotive-cluster.md#cluster-wide-pod-security-policy)
-for more details.
+>NOTE: Lokomotive uses a relatively restrictive Pod Security Policy by default. This policy
+>disallows running containers as root. Refer to the
+>[Pod Security Policy documentation](../concepts/securing-lokomotive-cluster.md#cluster-wide-pod-security-policy)
+>for more details.
 
 ## Cleanup
 
-To destroy the Lokomotive cluster, execute the following command:
+To destroy the cluster, execute the following command:
 
 ```console
 lokoctl cluster destroy --confirm
 ```
 
-You can safely delete the working directory created for this quickstart guide if you no longer
-require it.
+You can now safely delete the directory created for this guide if you no longer need it.
 
 ## Troubleshooting
 
-### Stuck at copy controller secrets
+In case the deployment fails, adding the `-v` flag to `lokoctl cluster apply` can provide useful
+information.
 
-If there is an execution error or no progress beyond the output provided below:
+### Stuck at "copy controller secrets"
 
 ```console
 ...
-module.packet-mycluster.null_resource.copy-controller-secrets: Still creating... (8m30s elapsed)
-module.packet-mycluster.null_resource.copy-controller-secrets: Still creating... (8m40s elapsed)
+module.packet-lokomotive-demo.null_resource.copy-controller-secrets: Still creating... (8m30s elapsed)
+module.packet-lokomotive-demo.null_resource.copy-controller-secrets: Still creating... (8m40s elapsed)
 ...
 ```
 
-The error probably happens because the `ssh_pubkeys` provided in the configuration is missing in the
-`ssh-agent`.
+In case the deployment process seems to hang at the `copy-controller-secrets` phase for a long
+time, check the following:
 
-To rectify the error, you need to:
-
-1. Follow the steps [to add the SSH key to the
-   ssh-agent](https://help.github.com/en/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent#adding-your-ssh-key-to-the-ssh-agent).
-2. Retry [Step 5](#step-5-create-lokomotive-cluster).
+- Verify the correct private SSH key was added to `ssh-agent`.
+- Verify that you can SSH into the created controller nodes from the machine running `lokoctl`.
 
 ### Packet provisioning failed
 
-For failed machine provisioning on Packet end, retry [Step 5](#step-5-create-lokomotive-cluster).
+Sometimes the provisioning of servers on Packet may fail. In this case, retrying the deployment by
+re-running `lokoctl cluster apply` may help.
```

Comment:
The user sees no machines on the Packet API, gets an error from `lokoctl` and gets an email from Packet saying "provisioning failed". IMO it's quite clear.

Reply:
Cool, thanks!

Reply:
Hmm, this seems duplicated with another comment? Closing, as it seems it is clear :)

Reply:
Well, thinking about it a little more... I don't think it is quite clear.

Lokomotive doesn't show any **clear** error, it can be **anything** that failed, AFAIK. Lokomotive error is not very indicative, at all, it just says that failed (it can be a network micro downtime, auth issues on AWS, etc.). Lokomotive, AFAIK, just don't show anything useful now.

You can check if servers are created in Packet web interface, but if they aren't what does it mean? It might be that provisioning failed or that there is insufficient capacity. If they need to check email to see if there was provisioning failure... it seems quite non-obvious.

IMHO, it is not quite clear. We can leave it like this and see what happens. Or try to be a little bit more clear (ask the user to check on packet web interface if servers are there, for example). Or create a follow-up issue to improve logging errors on lokomotive, because it is not possible to create reasonable guides with the current behavior (not showing what failed) and keep the guide as it is, as the error will be clearer soon.

But IMHO, it is not quite clear right now and with the documented steps.

Am I missing something?

Reply:
Let's keep this duplicate closed, just in case :)

Reply:
Same as in previous comments, I suggest we run with `-v` from the start, in which case if there is a provisioning error the user gets an explicit message from Terraform saying there was a provisioning error.

Reply:
Fully agree :)